halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/SpinnakerMonitoringDaemonService.java

```diff
@@ -97,6 +101,11 @@ private static String defaultFilterProfileName() {
     String profilePath = Paths.get(CONFIG_OUTPUT_PATH, profileName).toString();
 
     results.add(spinnakerMonitoringDaemonProfileFactory.getProfile(profileName, profilePath, deploymentConfiguration, endpoints));
+
+    profileName = monitoringLocalProfileName();
+    profilePath = Paths.get(CONFIG_OUTPUT_PATH, profileName).toString();
+    results.add(spinnakerMonitoringDaemonProfileFactory.getProfile(profileName, profilePath, deploymentConfiguration, endpoints));
```

Comment:
You're generating the same profile here as the non-local profile. Is this intentional?

Reply:
Removed.